src/documents/tutorial-userInterfaceOptions/UserInterfaceOptions.md

```diff
@@ -111,20 +118,20 @@ We'll use the `<script>` tag to link to the Infusion library:
 
 ```html
 <!-- The Infusion Library -->
-<script type="text/javascript" src="lib/infusion/infusion-custom.js"></script>
+<script type="text/javascript" src="lib/infusion/infusion-all.js"></script>
 ```
 
-<div class="infusion-docs-note"><strong>Note:</strong> that the <code>infusion-custom.js</code> file is a concatenation of all of the required JavaScript files and will be minified (i.e. all of the whitespace removed) if you've built the minified version. If so, it might be difficult to debug with. If you want to be able to debug the code, you might want to choose the "source" version when you create your Infusion bundle.</div>
+<div class="infusion-docs-note"><strong>Note:</strong> The <code>infusion-custom.js</code> file is a concatenation of all of the required JavaScript files and will be minified (i.e. all of the whitespace removed) if you've built the minified version. If so, it might be difficult to debug with. If you want to be able to debug the code, you might want to choose the "source" version when you create your Infusion bundle.</div>
```

Comment:
In the code block you mention `infusion-all.js` but in the note it's `infusion-custom.js`. I think you note could mention both or if just one, it should be consistent with what's in the code block.